symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js

```diff
@@ -0,0 +1,94 @@
+const empStatusAPI = '/api/v2/admin/employment-statuses';
+
+describe('Admin - Employment Status API ', function () {
+  beforeEach(function () {
+    cy.task('db:reset');
+    cy.fixture('chars').as('strings');
+    cy.fixture('user').then(({admin}) => {
+      cy.apiLogin(admin);
+    });
+  });
+
+  describe('POST /emp-Status', function () {
+    it('create a new employment status', function () {
+      cy.request('POST', empStatusAPI, {
+        name: this.strings.chars50.text,
+      }).then((response) => {
+        console.log(response);
```

Comment:
remove `console.log()`. run `yarn lint` for clean up.

Reply:
Done.